.github/workflows/doc_build.yml

```diff
@@ -0,0 +1,52 @@
+name: Docs
```

Comment:
Can you add .DS_Store to .gitignore, and any sphinx related build files, like _build/ build/?